rollup/rollup.config.js

```diff
@@ -0,0 +1,57 @@
+import copy from 'rollup-plugin-copy';
+import del from 'rollup-plugin-delete';
+import dynamicImportVars from '@rollup/plugin-dynamic-import-vars';
+import glob from 'glob-all';
+import { readFileSync } from 'fs';
+import replace from '@rollup/plugin-replace';
+import resolve from '@rollup/plugin-node-resolve';
+
+const { version } = JSON.parse(readFileSync('./package.json'));
+const buildDate = Intl.DateTimeFormat('en-CA', { timeZone: 'America/Toronto' }).format(new Date());
+
+const jsGlob = [
+	'@(src|demo)/**/*.js',
+	'./index.js',
+	'node_modules/@brightspace-ui/core/components/calendar/*.js',
+	'node_modules/@brightspace-ui/core/components/collapsible-panel/*.js',
+	'node_modules/@brightspace-ui/core/components/colors/*.js',
+	'node_modules/@brightspace-ui/core/components/demo/*.js',
+	'node_modules/@brightspace-ui/core/components/dropdown/*.js',
+	'node_modules/@brightspace-ui/core/components/icons/*.js',
+	'node_modules/@brightspace-ui/core/components/menu/*.js',
+	'node_modules/@brightspace-ui/core/components/typography/*.js',
```

Comment:
Don't love this, but we need to tell rollup to include these components, which are _only_ used in the demos. This is likely only a labs repo problem.

Reply:
Wondering if you could use the `html` plugin to point the input at all the `.html` files [like this](https://github.com/dlockhart/brightspace-ui-core-demo/blob/main/rollup.config.js) and then `node-resolve` would properly resolve the bare module imports?

Reply:
That works! I knew I was missing something. It creates a bunch of `inline-module` js files, but I'm not concerned just for these previews.